README.md

```diff
@@ -1,16 +1,89 @@
-# Title
+# Program With Expressions
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Write a program with conditional logic using evaluation expressions only
 
 ## Introduction
 
-## SWBAT 1
+You've traveled quite a road! You started off, from a conversational
+perspective, mute and unable to interact, and now you can converse with Ruby.
 
-## SWBAT 2
+Not only that, but you can ask Ruby to respond with some very sophisticated
+results! Let's prove it to ourselves! But take this as an important fact:
+
+***Programming is the art of helping a computer decide which expressions to
+evaluate and in which order.***
+
+Believe it or not, the est of your programming career will be learning to make
+your messages to Ruby, and other programmers, as clear and efficient as
+possible. That's the name of the game for this career ***and you're doing it
+now!***  
+
+## Write a Program with Conditional Logic Using Evaluation Expressions Only
+
+Recall when we introduced the ternary expression, we wrote this little program:
+
+```ruby
+likely_to_rain = true
+garment = likely_to_rain ? "galoshes" : "sun hat"
+```
+
+Let's expand on this idea and use our new-found knowledge of `String` to make
+the output more informative. You will see many of the expressions you learned in
+this module come into play. We're going to mix-and-match techniques to show you
+just how much you've learned!
+
+```ruby
+name = "Your name here"
+rain_percentage = 0.2
+temperature_in_c = 26
+
+likely_to_rain = rain_percentage > 0.30
+sun_is_dangerous = temperature_in_c >= 26
+
+"Hello, #{name}, with a rain chance of #{rain_percentage * 100}% and a temperature of #{temperature_in_c}C we recommend that you " + (likely_to_rain ? "take an umbrella" : "enjoy this rain-free day") +
+"#{sun_is_dangerous ? ' and watch out for heat stroke!' : ' and fine weather.'}"
+```
+
+This produces:
+
+```text
+=> "Hello Your name here with a rain chance of 20.0% and a\ntemperature of 26C we recommend that you enjoy this rain free day and watch out for heat stroke"
+```
+
+Change just a few variables and you get an entirely different scenario:
+
+```ruby
+name = "Byron the Poodle"
+rain_percentage = 0.8
+temperature_in_c = 26
+
+likely_to_rain = rain_percentage > 0.30
+sun_is_dangerous = temperature_in_c >= 26
+
+"Hello, #{name}, with a rain chance of #{rain_percentage * 100}% and a temperature of #{temperature_in_c}C we recommend that you " + (likely_to_rain ? "take an umbrella" : "enjoy this rain-free day") +
+"#{sun_is_dangerous ? ' and watch out for heat stroke!' : ' and fine weather.'}"
+```
+
+This produces:
+
+```text
+=> "Hello, Byron the Poodle, with a rain chance of 80.0% and a temperature of 26C we recommend that you take an umbrella and watch out for heat stroke!"
+```
```

Comment:
maybe adjust `and find weather.` since combining it with 'take an umbrella' doesn't read right:

"Hello, Byron the Poodle, with a rain chance of 80.0% and a temperature of 20C we recommend that you take an umbrella and find weather."